addons/mail/views/discuss/discuss_notification_settings_views.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+<odoo>
+    <data>
+        <record id="discuss_notification_settings_action" model="ir.actions.client">
+            <field name="name">Notification Settings</field>
+            <field name="tag">discuss.notification_settings_action</field>
+            <field name="target">new</field>
+            <field name="context">{'dialog_size': 'medium', 'footer': false}</field>
+        </record>
```

Comment:
Confusing split, if the menuitem is in mail (and not discuss folder) this should be too, and so most of the feature too.

The only part that should be in discuss folder should be the channel specific stuff.